source-browser.html

```diff
@@ -182,6 +193,32 @@ <h1 id="loading" class="m1 center display-none">⌛</h1>
       }
 
       $(document).ready(() => {
+        $(window).on('scroll resize', function(){
+            const top = $(window).scrollTop();
+            const bottom = $(window).scrollTop() + $(window).height();
+            TOPICS.forEach(heading => {
```

Comment:
I think append can take a list, right? In that case, use a map to create the list and append that.

Reply:
Which append are you referring to?  
 `$('.li').append(` ?

Reply:
From what I understand, that area of the code doesn't need appending.